src/app/tests/suites/TestGroupMessaging.yaml

```diff
@@ -32,13 +32,13 @@ tests:
       attribute: "location"
       groupId: "1234"
       arguments:
-          value: "us"
+          value: "XX"
 
     - label: "Read back Attribute"
       command: "readAttribute"
       attribute: "location"
       response:
-          value: "us"
+          value: "XX"
```

Comment:
And this should be reading back the value that got written above....

Reply:
The CI always complain the 'XX' is expected, and from the debug log it seems "Group Write Attribute" does not trigger any write to the ConfigurationMgr, but "Write Attribute" did.

Reply:
Right now, on tip, the "Group Write Attribute" thing works....  Does that answer the question or not quite?

Reply:
Yes, the Group Write Attribute / Read back Attribute case passed, but Restore initial location value / Read back Attribute case still failed. Need to check why "" country code still returned

Reply:
The problem is all of those attributes in basic cluster are only set once at the beginning based on the configured values, after that, the read and write are conducted from and to the attribute store directly. 

After 'Restore initial location value' test case, 'Read back Attribute ' test case read the value from the attribute store instead of platform API. That is why the return value is empty.

Reply:
We need to intercept all read/write API for writable attributes, I implemented the API for location in this PR to fix the yaml test. Other writable attributes is tracked in https://github.com/project-chip/connectedhomeip/issues/13130

Reply:
You mean intercept writes for all the cases that are intercepting read, right?